mlir/test/Conversion/ArithToEmitC/arith-to-emitc.mlir

```diff
@@ -717,3 +717,36 @@ func.func @arith_index_castui(%arg0: i32) -> i32 {
 
   return %int : i32
 }
+
+// -----
+
+func.func @arith_divui_remui(%arg0: i32, %arg1: i32) -> i32 {
```

Comment:
Why is there a new divui/remui test included here?

Reply:
Mmh, `git reset --soft` I guess.